src/components/molecules/PlanRoom/RoomTables/Table/index.tsx

```diff
@@ -0,0 +1,182 @@
+import { useCallback, useState, SetStateAction } from 'react';
+import { Box, Textarea } from '@chakra-ui/react';
+import update from 'immutability-helper';
+import { useDrop, XYCoord, useDrag } from 'react-dnd';
+import ItemTypes from 'components/molecules/PlanRoom/RoomTables/TableDNDTypes';
+import { TypeItem, TableType, TablesType } from 'components/molecules/PlanRoom/RoomTables/TableComponentsType';
+import SaveBoard from 'components/molecules/PlanRoom/SaveRoomTables/SaveBoard';
+import { theme } from 'others/theme';
+
+let changeValueTable: Function;
+let changeValueTableBorderRadius: Function;
+let addNewTable: Function;
+let save: Function;
+
+const Items = () => {
```

Comment:
Maybe rename that component to something like `TableList`?

Reply:
I agree with you. I will change it to your proposed